src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -164,7 +169,14 @@ public static function fromDsn(string $dsn, array $redisOptions = [], \Redis $re
         }
 
         $tls = false;
+        if ('rediss' === $scheme) {
+            $tls = true;
+        }
         if (\array_key_exists('tls', $redisOptions)) {
+            if ($tls) {
+                throw new LogicException('"tls" parameter provided while TLS was already enabled with rediss:// scheme.');
+            }
+            trigger_deprecation('symfony/redis-messenger', '5.3', 'Providing "tls" parameter is deprecated, use "rediss://" DSN scheme instead');
```

Comment:
Please add a @legacy test case for this deprecation

Reply:
Sorry, what exactly do you mean? To set the `@group legacy` in the annotation for existing tests that use the `tls` parameter or, something else?

Reply:
Yes

Reply:
Just to confirm that you were referring to this? https://github.com/symfony/symfony/pull/39607/commits/c7508fe20370fc7d1423f3cfddfc1a4eb9bf12d2

Reply:
Yes thanks